examples/flutter_gallery/lib/demo/cupertino/cupertino_alert_demo.dart

```diff
@@ -201,6 +110,105 @@ class _CupertinoAlertDemoState extends State<CupertinoAlertDemo> {
       ),
     );
   }
+
+  void _onAlertPress(BuildContext context) {
+    showDemoDialog(
+      context: context,
+      child: CupertinoAlertDialog(
+        title: const Text('Discard draft?'),
+        actions: <Widget>[
+          CupertinoDialogAction(
+            child: const Text('Discard'),
+            isDestructiveAction: true,
+            onPressed: () {
```

Comment:
Nit: This and similar others below could be one liners.
```suggestion
            onPressed: () => ...
```

Reply:
Yes. I only moved the code to make the review easier.
I will update the PR.